include/host_checks/DomainNamesConnection.h

```diff
@@ -0,0 +1,47 @@
+/*
+ *
+ * (C) 2013-21 - ntop.org
+ *
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ *
+ */
+
+#ifndef _DOMAIN_NAMES_CONNECTION_H_
+#define _DOMAIN_NAMES_CONNECTION_H_
+
+#include "ntop_includes.h"
+
+class DomainNamesConnection : public HostCheck {
```

Comment:
- Name this class `DomainNamesContacts` to stay consistent with other classes `DNSServerContacts`, `SMTPServerContacts`, and so on.
- Make `DomainNamesContacts` son of `ServerContacts` (possibly rename the latter in `Contacts`).
- Also rename the corresponding enums and typedefs
- Do the very same things described above for the corresponding alert class `DomainNamesConnectionAlert`.